nodejs/indy/src/wallet/index.js

```diff
@@ -11,15 +11,19 @@ exports.get = async function() {
     return wallet;
 };
 
+//TODO: replace whatever with real user password
 exports.setup = async function () {
     try {
-        await sdk.createWallet(config.poolName, config.walletName);
+        //await sdk.createWallet(config.poolName, config.walletName, "default", null, {"key": "whatever"});
```

Comment:
When I try this commented out version, I just get a `Bus Error` but I don't know that line 18's syntax is supported.

Reply:
Have you gotten this part working?  We need to specify a pool name somewhere don't we?

Reply:
i wrote in indy-sdk channel, that we need a new version of indy-sdk npm package here: https://www.npmjs.com/package/indy-sdk

Reply:
the new version of createWallet has only 2 params:
[https://github.com/hyperledger/indy-sdk/blob/master/libindy/src/api/wallet.rs](https://github.com/hyperledger/indy-sdk/blob/master/libindy/src/api/wallet.rs)

Reply:
the second pull request changes Dockerfile so that the indy-sdk will be downloaded, libindy compiled and indy-sdk wrapper for nodejs will be linked/installed in indy-agent

Reply:
ok, your right, it"s a feature of indy-sdk > 1.5

Reply:
we can merge this PR after tagging agent

Reply:
Ok. I will take a look at the other pull request for now.